modules/3d-tiles/wip/tile-3d-header.md

```diff
@@ -0,0 +1,238 @@
+# Tile3DHeader
```

Comment:
do we still need this `wip`?

Reply:
> do we still need this wip?

Doing a full doc update would be too big an effort right now, so I wanted to focus on removing any text in the documentation that is not clearly correct/up-to-date. Would rather have too little docs than misleading docs.

My thinking is that by keeping the original text in `wip` we don't have to rewrite these docs when we continue work on `Tileset3D` in the next release.